vestapol/web_resources/base_resource.py

```diff
@@ -83,7 +81,7 @@ def get_hive_path(data_path: List[Tuple[str, str]]):
 
         return hive_path
 
-    def get_pathname(self, data_path: List[Tuple[str, str]] = None):
+    def get_pathname(self, data_path: List[Tuple[str, Any]] = None) -> str:
```

Comment:
Question: in practice, get_pathname() returns a `pathlib.Path` object (since `get_response_root()` returns a path), though that object would support any methods that`str` supports. How would we need to annotate things such that the mypy assertion "`get_pathname()` returns a `str`" would fail? Would we just need to annotate `get_response_root()` with ` -> pathlib.Path`? Maybe I'm a little surprised that mypy isn't automatically inferring the return value of `get_response_root()`.

Reply:
it might be, i was adding return types just as helpful hints. i think i just made a mistake on this one